hadoop-tools/hadoop-azure/src/site/markdown/testing_azure.md

```diff
@@ -872,7 +872,7 @@ To run CheckAccess test cases you must register an app with no RBAC and set
 the following configurations.
 ```xml
 <!--===========================   FOR CheckAccess =========================-->
-<!-- To run ABFS CheckAccess SAS tests, you must register an app, with no role
+<!-- To run ABFS CheckAccess tests, you must register an app, with no role
```

Comment:
can you pull these instructions up from the XML into the text including URLs becoming links. I will be using these instructions to get my token